scripts/ensure-env.js

```diff
@@ -1,5 +1,5 @@
 var fs = require("fs");
 var env = require("path").join(__dirname, "..", ".env");
 
-if (fs.existsSync(env) || process.env.NODE_ENV == "production") process.exit(0);
+if (fs.existsSync(env) || !(typeof process.env.NODE_ENV === 'undefined')) process.exit(0);
```

Comment:
small nit: ` !(typeof process.env.NODE_ENV === 'undefined')` => `typeof process.env.NODE_ENV !== 'undefined'`, just to remove some parentheses.